codalab/lib/bundle_cli.py

```diff
@@ -2957,7 +2957,7 @@ def do_wedit_command(self, args):
                 owner = client.fetch('users', args.owner_spec)
                 info['owner'] = JsonApiRelationship('users', owner['id'])
             if args.freeze:
-                info['frozen'] = datetime.datetime.utcnow().isoformat()
+                info['frozen'] = datetime.datetime.utcnow()
```

Comment:
`datetime.datetime.utcnow()` will return a datetime object, but with `isoformat()`, it will return a string representing the date in ISO 8601 format. Maybe we should keep the `isoformat()`?

Reply:
The reason why I changed it is because https://github.com/codalab/codalab-worksheets/blob/179e117b3ee4b4a51013fa4c6509e1d83ec58cac/codalab/rest/worksheets.py#L375-L378 previously we used .now(), which still gives a datetime. Should we make them consistent?

Reply:
Yes. Keep them consistent would be better. And it also depends on what we want to save in `info[frozen]`

Reply:
Do you know if we want a string or a datetime?

Reply:
Right now `info[frozen]` is not really used anywhere. metadata is also separate from the value.